swap/src/asb/config.rs

```diff
@@ -58,7 +58,7 @@ impl GetDefaults for Mainnet {
             data_dir: default_asb_data_dir()?.join("mainnet"),
             listen_address_tcp: Multiaddr::from_str("/ip4/0.0.0.0/tcp/9939")?,
             listen_address_ws: Multiaddr::from_str("/ip4/0.0.0.0/tcp/9940/ws")?,
-            electrum_rpc_url: Url::parse("ssl://electrum.blockstream.info:50002")?,
+            electrum_rpc_url: Url::parse("ssl://blockstream.info:700")?,
```

Comment:
Does the old url no longer work?

Reply:
It work but it is very unstable. That's why I propose to change to this electrum node referenced in the official blog of Blockstream : https://blog.blockstream.com/en-esplora-and-other-alternatives-to-electrumx/